### PR TITLE
GSoC 2018: Final Report - Rubyplot GR

### PR DESCRIPTION
**Pull request by pgtgrly**

### CODE REVIEW

File: source/_posts/2019-01-23-gsoc-2018-rubyplot -gr- a-scientific-plotting-library-for-ruby-built-on-gr.markdown [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+comments: true 
+author: Pranav Garg 
+categories: [GSoC, GSoC 2018, RubyPlot, Data Visualization, Ruby, Matplotlib, GR Framework]
+---
+
+### 1. About
+
+For my summer of code Project I decided to create a plotting library.
+
+From the scratch.
```
Comment (v0dro) [inline source/_posts/2019-01-23-gsoc-2018-rubyplot -gr- a-scientific-plotting-library-for-ruby-built-on-gr.markdown]: From scratch. In Ruby.

File: source/_posts/2019-01-23-gsoc-2018-rubyplot -gr- a-scientific-plotting-library-for-ruby-built-on-gr.markdown [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+in Ruby.
+
+### 2. Application
+
+The GSoC 2018 application can be found [here](https://github.com/pgtgrly/Cairo_Graph/wiki/Google-Summer-of-Code-2018-Application).
```
Comment (v0dro) [inline source/_posts/2019-01-23-gsoc-2018-rubyplot -gr- a-scientific-plotting-library-for-ruby-built-on-gr.markdown:20]: Please also add your proposal to this wiki: https://github.com/SciRuby/sciruby/wiki/Successful-GSOC-proposals

**Review by v0dro — CHANGES_REQUESTED** [review]
Some changes needed but otherwise looks good.

File: source/_posts/2019-01-23-gsoc-2018-rubyplot -gr- a-scientific-plotting-library-for-ruby-built-on-gr.markdown (hunk 2 above)
Comment (pgtgrly) [reply source/_posts/2019-01-23-gsoc-2018-rubyplot -gr- a-scientific-plotting-library-for-ruby-built-on-gr.markdown:20]: Done

**Comment by prasunanand** [comment]
@pgtgrly Please look into it. Need to merge by EOTD.

**Pull request merged**